docs/get-started/viam.md

```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
 images: ["/general/understand.png"]
 ---
 
-Viam is a complete software platform that supports every step of your {{< glossary_tooltip term_id="smart-machine" text="smart machine">}} development lifecycle.
+Viam is a software platform that makes it easy to work with hardware and software.
```

Comment:
I like this sentence. It's easy to read and gives straightforward overarching context.